ocfweb/docs/docs/staff/startertasks.md

```diff
@@ -31,6 +31,25 @@ See http://www.irchelp.org/clients/ for more recommendations.
 
 Once you’ve joined IRC, pick any channel (#rebuild, #henlo, etc) and say hi!
 
+## Get familiar with the command line
+All of [[our servers|doc staff/backend/servers]] run Linux, and we interact with
+them over the command line. There are a lot of online resources about using the
+Linux command line, so if you're confused about something, try Googling it!
+
+Here are a few commands you will want to know about:
+- `cd` (change directory): move from one directory (folder) to another
+- `ls` (list): list the files in the current directory
+- `cat` (concatenate): print file contents
+- `less`: read long files
+- `mv` (move): move or rename files
+- `cp` (copy): copy files
+- `rm` (remove): delete files or directories
```

Comment:
This is true, but I wasn't where to guide people as far as learning basic commands. It's not reasonable to just say "log in and learn some commands!" without further guidance, but if there's a good resource to link to or something I agree that it would be better than this list.